recipe/meta.yaml

```diff
@@ -13,12 +13,18 @@ source:
 
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed --install-option="--conda" .
+  script: |
+     python -m pip install --no-deps --ignore-installed --install-option="--conda" . ;
+     jupyter-nbextension install nglview --py --sys-prefix ;
+     jupyter-nbextension enable nglview --py --sys-prefix
```

Comment:
why does this help user installing nglview? Does this only happen during the build process, not the install process?

Reply:
@marscher I am still confused here. Saying you run the build process on your computer, the  `jupyter-nbextension install` and `jupyter-nbextension enable` only happen during the build process in your computer. If I install the built package to my computer, how does the `enable` happen?

Reply:
The enable happens via an activation file stored in the installation prefix (as well as the JS files). The file(s) is created during the build process. This way the extension is also properly uninstalled.
Does this make sense to you now?

Reply:
make sense to me now.

May be we already do that in setup.py:
https://github.com/arose/nglview/blob/master/setup.py#L40

Reply:
But it's no hurt to do it again here in this recipe to be sure.